builds/jenkins-build-pack-api/Jenkinsfile

```diff
@@ -84,6 +84,18 @@ node() {
 		error "Failed to fetch service metadata from catalog"
 	}
 
+  if(!config.region) {
+    config.region = configLoader.AWS.DEFAULTS.REGION
+  }
+
+  if(!config.accountId) {
+    config.accountId = configLoader.AWS.DEFAULTS.ACCOUNT_ID
+  }
+
+  if(!config.provider) {
+    service_config.provider = configLoader.JAZZ.DEFAULTS.PROVIDER
```

Comment:
where is this variable used?

Reply:
Currently it is using , while we are sending events. I am not seeing any other hard coded values for provider.